tools/devtools/templates/new-in-devtools.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Sorry, didn't think about it earlier, also just an idea: as this is essentially a template you could also use any of the template languages that 11ty ships with. Nunjucks for example. Pro: you would not have to deal with RegEx patterns and String.replace, Con: you would need to wrap things like `{% include %}` with `{% raw %}`. Your call 🙂